cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/task/RunTestsTask.java

```diff
@@ -49,6 +49,12 @@ public void execute(BuildContext buildContext) {
         // as "." are ignored. This is to be consistent with the "ballerina test" command which only executes tests
         // in packages.
         for (BLangPackage bLangPackage : moduleBirMap) {
+            PackageID packageID = bLangPackage.packageID;
+
+            if (!buildContext.moduleDependencyPathMap.containsKey(packageID)) {
```

Comment:
Is there are cases when the map doesn't contain packageID

Reply:
Yes, there was a case when i tested and it threw NPE